multegula.go

```diff
@@ -118,6 +118,22 @@ func getMessage(nodes []messagePasser.Node, localNodeName string) messagePasser.
 	return messagePasser.Message{Source: localNodeName, Destination: dest, Content: content, Kind: kind}
 }
 
+/* receive message from PyBridge and send to messagePasser */
+func sendToMessagePasser() {
+    for {
+        message := bridges.ReceiveFromPyBridge()
+        messagePasser.Send(message)
+    } 
+}
+
+/* receive message from MessagePasser and send to PyBridge */
+func receiveFromMessagePasser() {
+    for {
+        message := messagePasser.BlockReceive()
```

Comment:
BlockReceive will circumvent the `receiveQueue` by just pulling messages out of the `receiveChannel`! I don't recommend doing that as it will result in duplicate/out of order messages.